packages/node_modules/@ciscospark/plugin-phone/test/integration/lib/cmr.js

```diff
@@ -14,18 +14,24 @@ export default class CMR {
       return Promise.reject(new Error('spark is required'));
     }
 
+    let resourceType = 'CMR_3';
+    // Set resource type for known integration environment.
+    if (process.env.WHISTLER_API_SERVICE_URL === 'https://whistler.onint.ciscospark.com/api/v1') {
+      resourceType = 'CMR_4';
+    }
+
     console.log('reserving cmr');
     return spark.request({
       method: 'POST',
-      uri: 'https://whistler.onint.ciscospark.com/api/v1/reservations',
+      uri: `${process.env.WHISTLER_API_SERVICE_URL || 'https://whistler-prod.onint.ciscospark.com/api/v1'}/reservations`,
       headers: {
         authorization: `Bearer ${spark.credentials.supertoken.access_token}`,
         'cisco-no-http-redirect': null,
         'spark-user-agent': null,
         trackingid: `ITCLIENT_${uuid.v4()}_0_imi:true`
       },
       body: {
-        resourceType: 'CMR_3',
+        resourceType,
```

Comment:
I think you can make this a ternary instead of doing the logic above and having to reassign resourceType
This is more of a syntax nitpick than anything else
Will approve regardless

Reply:
Thanks, Taymoor - I did consider this, but the line looked a little long to read easily.